README.md
akvo-web
========

Akvo.org web site and blog

[![Stories in Ready](https://badge.waffle.io/akvo/akvo-web.png?label=ready&title=Ready)](http://waffle.io/akvo/akvo-web)


RELEASE_NOTES.md

Tuesday, 27 June @lynngre

--------
Akvo.org v1.2.14 WordPress update & Page Builder addition
---

## Bug fixes & changes

### Wordpress 4.8

WordPress update.

Github issue: #1164 

### SiteOrigin page builder & widget plugins

Page builder & widget plugins added, code for support for page builder use.

Github issue: #1160 


Wednesday 17 May 2017, @lynngre

--------
Akvo.org v1.2.13 WordPress security & plugins update
---

## Bug fixes & changes

### Wordpress 4.7.5

WordPress security update.

Github issue: #1149 #1153 #1156 

### Robots.txt & sitemap.xml

Added plugin to allow setting of text in Robots.txt
Added plugin to allow creation of a sitemap.xml for Google search engine

Github issue: #1150 #1135 

### Small change to team page

Team page header updated.

Github issue: #1147 

###  Backend code for the new Akvo Sites page

Bring layout inline with other product pages

Github issue:  #1002 

### New partner page

Beginnings of the new partner page area in the backend.

Github issue: #1086 



Thursday 28 April 2016, @lynngre

--------
Akvo.org v1.2.5 Wongi waverider
---

## New Features

###  Akvo Dash 

New page (Akvo Dash) for Research & Development area.
Ability to update logo.

Github issue:  #955 #922 #948 

###  Heroes series

New template to allow for listing of hero blog posts .  Update list manually backend. Url to be added to end of each hero post.

Github issue:  #749 

### Menu item level

Additional level to the menu item.

Github issue:  #943 


## Bug fixes & changes

###  Working at Akvo

Custom field for job posting required updating to wysiwyg editor.

Github issue: #945 

### RSR product page

Layout changes.

Github issue:  #939 




Thursday 20 August 2015, @lynngre

--------
Akvo.org v1.2.0A Retrocognition Rhubarb
---


## Hotfix

### Jagged left edge on carousel

Border on carousel is not displaying correctly in Safari & Chrome for some visitors.

Github issues:  #740 

### Force WordPress to load a new CSS file

Using 'version control' type numbering of css files to push visitors' cache to refresh.

Github issues:  #734 

### Buttons on about us

Buttons towards bottom of the About-Us page shouldn't change cover when hovered over.

Github issues:  #731 

### New Akvo Sites logo

Added on the Akvo Sites product page.

Github issues:  #729 


Tuesday 18 August 2015, @lynngre

--------
Akvo.org v1.2.0 Retrocognition Rhubarb
---

## New & noteworthy

We've given our homepage a facelift!  

The site has been updated to explain as clearly as possible what we do. We’re keen to help existing and new partners use data to improve decision making and are in [Stockholm](http://akvo.org/blog/akvo-stockholm9-team-world-water-week-2015/ "Stockholm") next week to engage with people on exactly that. 

In trying to describe Akvo 'in a nutshell' we've also updated our [About Us](http://akvo.org/about-us/ "About us") page in line with the above.

We have a shiny new [Akvo Tools](http://akvo.org/products/ "Akvo tools") page too.

## New features

###  About us

Updated to reflect data to decision.  Customisable by Comms team.

Github issue: #673 

###  Homepage

Updated to reflect data to decision.  Includes new header, different content boxes of various sizes.

Github issue: #632  

###  Akvo Tools

Product page with only four product buckets,  is replaced by the new Akvo tools page with customisable boxes of various sizes.

Github issue: #672   


## Bug fixes & changes

###  Footer

Footer has been updated.

Github issue: #703 

### Simple tooltips replacement

Plugin was creating errors on the page.  This has been replaced with our own code.  Use of Simple Tooltips is no longer required.

Github issue:  #634 

### Remove header image tag from about us

Github issue:  #723 


Wednesday 10 June 2015, @lynngre

--------
Akvo.org WordPress v4.2.2
---

## Security release

https://codex.wordpress.org/Version_4.2.2 latest release security & maintenance.


Thursday 9 July 2015, @lynngre

--------
Akvo.org v1.1.16  Quantum Quandong HOT FIX
---

An update to the jquery library in v1.1.16 broke the layout of our partner and team pages.

The problem:  #681 

The fix:  #692 


Tuesday 7 July 2015, @lynngre

--------
Akvo.org v1.1.16  Quantum Quandong
---

## New & noteworthy

We have lost our old footer!  We now have a shiny new one.

This release also contains part of the backend for 'Akvo in a Nutshell' (the new 'About us' page) and WordPress plugin updates.


## New features

###  Footer

Footer has been modernised & decluttered.

Github issue: #633

###  'Akvo in a nutshell' backend

Allows for the future creation of a new 'About us' page heavily dependent on the new marketing approach around our products.

Github issue: #673 


## Bug fixes & changes

### Wordpress plugin updates

Various.

Github issue:  #631 

### Overlapping map pins

Corrected on 'Contact us' page.

Github issue:  #626 

### Various small fixes

Github issue:  #685 #674 #669  

Wednesday 10 June 2015, @lynngre

--------
Akvo.org Wordpress v4.2.2
---

## Security release

https://codex.wordpress.org/Version_4.2.2 latest release security & maintenance.


Tuesday 9 June 2015, @lynngre

--------
Akvo.org v1.1.15 Prehensile Prickly Pear

code/wp-content/w3tc-config/master.php
<?php

return array(
	'version' => '0.9.4.1',
	'cluster.messagebus.debug' => false,
	'cluster.messagebus.enabled' => false,
	'cluster.messagebus.sns.region' => '',
	'cluster.messagebus.sns.api_key' => '',
	'cluster.messagebus.sns.api_secret' => '',
	'cluster.messagebus.sns.topic_arn' => '',
	'dbcache.debug' => false,
	'dbcache.enabled' => false,
	'dbcache.engine' => 'file',
	'dbcache.file.gc' => 3600,
	'dbcache.file.locking' => false,
	'dbcache.lifetime' => 180,
	'dbcache.memcached.persistant' => true,
	'dbcache.memcached.servers' => array(
		0 => '127.0.0.1:11211',
	),
	'dbcache.reject.cookie' => array(
	),
	'dbcache.reject.logged' => true,
	'dbcache.reject.sql' => array(
		0 => 'gdsr_',
		1 => 'wp_rg_',
	),
	'dbcache.reject.uri' => array(
	),
	'dbcache.reject.words' => array(
		0 => '^\\s*insert\\b',
		1 => '^\\s*delete\\b',
		2 => '^\\s*update\\b',
		3 => '^\\s*replace\\b',
		4 => '^\\s*create\\b',
		5 => '^\\s*alter\\b',
		6 => '^\\s*show\\b',
		7 => '^\\s*set\\b',
		8 => '\\bautoload\\s+=\\s+\'yes\'',
		9 => '\\bsql_calc_found_rows\\b',
		10 => '\\bfound_rows\\(\\)',
		11 => '\\bw3tc_request_data\\b',
	),
	'objectcache.enabled' => true,
	'objectcache.debug' => true,
	'objectcache.engine' => 'file',
	'objectcache.file.gc' => 3600,
	'objectcache.file.locking' => false,
	'objectcache.memcached.servers' => array(
		0 => '127.0.0.1:11211',
	),
	'objectcache.memcached.persistant' => true,
	'objectcache.groups.global' => array(
		0 => 'users',
		1 => 'userlogins',
		2 => 'usermeta',
		3 => 'user_meta',
		4 => 'site-transient',
		5 => 'site-options',
		6 => 'site-lookup',
		7 => 'blog-lookup',
		8 => 'blog-details',
		9 => 'rss',
		10 => 'global-posts',
	),
	'objectcache.groups.nonpersistent' => array(
		0 => 'comment',
		1 => 'counts',
		2 => 'plugins',
	),
	'objectcache.lifetime' => 180,
	'objectcache.purge.all' => false,
	'fragmentcache.enabled' => false,
	'fragmentcache.debug' => false,
	'fragmentcache.engine' => 'file',
	'fragmentcache.file.gc' => 3600,
	'fragmentcache.file.locking' => false,
	'fragmentcache.memcached.servers' => array(
		0 => '127.0.0.1:11211',
	),
	'fragmentcache.memcached.persistant' => true,
	'fragmentcache.lifetime' => 180,
	'fragmentcache.groups' => array(
	),
	'pgcache.enabled' => true,
	'pgcache.comment_cookie_ttl' => 1800,
	'pgcache.debug' => true,
	'pgcache.engine' => 'file_generic',
	'pgcache.file.gc' => 3600,
	'pgcache.file.nfs' => false,
	'pgcache.file.locking' => false,
	'pgcache.lifetime' => 3600,
	'pgcache.memcached.servers' => array(
		0 => '127.0.0.1:11211',
	),
	'pgcache.memcached.persistant' => true,
	'pgcache.check.domain' => true,
	'pgcache.cache.query' => true,
	'pgcache.cache.home' => true,
	'pgcache.cache.feed' => true,
	'pgcache.cache.nginx_handle_xml' => false,
	'pgcache.cache.ssl' => false,
	'pgcache.cache.404' => false,
	'pgcache.cache.flush' => false,
	'pgcache.cache.headers' => array(
		0 => 'Last-Modified',
		1 => 'Content-Type',
		2 => 'X-Pingback',
		3 => 'P3P',
	),
	'pgcache.compatibility' => false,
	'pgcache.remove_charset' => false,
	'pgcache.accept.uri' => array(
		0 => 'sitemap(_index)?\\.xml(\\.gz)?',
		1 => '[a-z0-9_\\-]+-sitemap([0-9]+)?\\.xml(\\.gz)?',
	),
	'pgcache.accept.files' => array(
		0 => 'wp-comments-popup.php',
		1 => 'wp-links-opml.php',
		2 => 'wp-locations.php',
	),
	'pgcache.accept.qs' => array(
		0 => '',
	),
	'pgcache.reject.front_page' => false,
	'pgcache.reject.logged' => false,
	'pgcache.reject.logged_roles' => false,
	'pgcache.reject.roles' => array(
		0 => '',
	),
	'pgcache.reject.uri' => array(
		0 => 'wp-.*\\.php',
		1 => 'index\\.php',
	),
	'pgcache.reject.ua' => array(
		0 => '',
	),
	'pgcache.reject.cookie' => array(
		0 => 'wptouch_switch_toggle',
	),
	'pgcache.reject.request_head' => false,
	'pgcache.purge.front_page' => true,
	'pgcache.purge.home' => true,
	'pgcache.purge.post' => true,
	'pgcache.purge.comments' => false,
	'pgcache.purge.author' => false,
	'pgcache.purge.terms' => false,
	'pgcache.purge.archive.daily' => false,
	'pgcache.purge.archive.monthly' => false,
	'pgcache.purge.archive.yearly' => false,
	'pgcache.purge.feed.blog' => true,
	'pgcache.purge.feed.comments' => false,
	'pgcache.purge.feed.author' => false,
	'pgcache.purge.feed.terms' => false,
	'pgcache.purge.feed.types' => array(
		0 => 'rss2',
	),
	'pgcache.purge.postpages_limit' => 15,
	'pgcache.purge.pages' => array(
		0 => '',
	),
	'pgcache.purge.sitemap_regex' => '([a-z0-9_\\-]*?)sitemap([a-z0-9_\\-]*)?\\.xml',
	'pgcache.prime.enabled' => true,
	'pgcache.prime.interval' => 1000,
	'pgcache.prime.limit' => 15,
	'pgcache.prime.sitemap' => '',
	'pgcache.prime.post.enabled' => true,
	'minify.enabled' => true,
	'minify.auto' => true,
	'minify.debug' => true,
	'minify.engine' => 'file',
	'minify.file.gc' => 86400,
	'minify.file.nfs' => false,
	'minify.file.locking' => false,
	'minify.memcached.servers' => array(
		0 => '127.0.0.1:11211',
	),
	'minify.memcached.persistant' => true,
	'minify.rewrite' => false,
	'minify.options' => array(
	),
	'minify.symlinks' => array(
	),
	'minify.lifetime' => 86400,
	'minify.upload' => true,
	'minify.html.enable' => false,
	'minify.html.engine' => 'html',
	'minify.html.reject.feed' => false,
	'minify.html.inline.css' => false,
	'minify.html.inline.js' => false,
	'minify.html.strip.crlf' => false,
	'minify.html.comments.ignore' => array(
		0 => 'google_ad_',
		1 => 'RSPEAK_',
	),
	'minify.css.enable' => false,
	'minify.css.engine' => 'css',
	'minify.css.combine' => false,
	'minify.css.strip.comments' => false,
	'minify.css.strip.crlf' => false,
	'minify.css.imports' => '',
	'minify.css.groups' => array(
	),
	'minify.js.enable' => false,
	'minify.js.engine' => 'js',
	'minify.js.combine.header' => false,
	'minify.js.header.embed_type' => 'blocking',
	'minify.js.combine.body' => false,
	'minify.js.body.embed_type' => 'blocking',
	'minify.js.combine.footer' => false,
	'minify.js.footer.embed_type' => 'blocking',
	'minify.js.strip.comments' => false,
	'minify.js.strip.crlf' => false,
	'minify.js.groups' => array(
	),
	'minify.yuijs.path.java' => 'java',
	'minify.yuijs.path.jar' => 'yuicompressor.jar',
	'minify.yuijs.options.line-break' => 5000,
	'minify.yuijs.options.nomunge' => false,
	'minify.yuijs.options.preserve-semi' => false,
	'minify.yuijs.options.disable-optimizations' => false,
	'minify.yuicss.path.java' => 'java',
	'minify.yuicss.path.jar' => 'yuicompressor.jar',
	'minify.yuicss.options.line-break' => 5000,
	'minify.ccjs.path.java' => 'java',
	'minify.ccjs.path.jar' => 'compiler.jar',
	'minify.ccjs.options.compilation_level' => 'SIMPLE_OPTIMIZATIONS',
	'minify.ccjs.options.formatting' => '',
	'minify.csstidy.options.remove_bslash' => true,
	'minify.csstidy.options.compress_colors' => true,
	'minify.csstidy.options.compress_font-weight' => true,
	'minify.csstidy.options.lowercase_s' => false,
	'minify.csstidy.options.optimise_shorthands' => 1,
	'minify.csstidy.options.remove_last_;' => false,
	'minify.csstidy.options.case_properties' => 1,
	'minify.csstidy.options.sort_properties' => false,
	'minify.csstidy.options.sort_selectors' => false,
	'minify.csstidy.options.merge_selectors' => 2,
	'minify.csstidy.options.discard_invalid_properties' => false,
	'minify.csstidy.options.css_level' => 'CSS2.1',
	'minify.csstidy.options.preserve_css' => false,
	'minify.csstidy.options.timestamp' => false,
	'minify.csstidy.options.template' => 'default',
	'minify.htmltidy.options.clean' => false,
	'minify.htmltidy.options.hide-comments' => true,
	'minify.htmltidy.options.wrap' => 0,
	'minify.reject.logged' => false,
	'minify.reject.ua' => array(
		0 => '',
	),
	'minify.reject.uri' => array(
		0 => '',
	),
	'minify.reject.files.js' => array(
		0 => '',
	),
	'minify.reject.files.css' => array(
		0 => '',
	),
	'minify.cache.files' => array(
		0 => 'https://ajax.googleapis.com',
	),
	'cdn.enabled' => false,
	'cdn.debug' => false,
	'cdn.engine' => 'maxcdn',
	'cdn.uploads.enable' => true,
	'cdn.includes.enable' => true,
	'cdn.includes.files' => '*.css;*.js;*.gif;*.png;*.jpg;*.xml',
	'cdn.theme.enable' => true,
	'cdn.theme.files' => '*.css;*.js;*.gif;*.png;*.jpg;*.ico;*.ttf;*.otf,*.woff',
	'cdn.minify.enable' => true,
	'cdn.custom.enable' => true,
	'cdn.custom.files' => array(
		0 => 'favicon.ico',
		1 => '{wp_content_dir}/gallery/*',
		2 => '{wp_content_dir}/uploads/avatars/*',
		3 => '{plugins_dir}/wordpress-seo/css/xml-sitemap.xsl',
		4 => '{plugins_dir}/wp-minify/min*',
		5 => '{plugins_dir}/*.js',
		6 => '{plugins_dir}/*.css',
		7 => '{plugins_dir}/*.gif',
		8 => '{plugins_dir}/*.jpg',
		9 => '{plugins_dir}/*.png',
	),
	'cdn.import.external' => false,
	'cdn.import.files' => '',
	'cdn.queue.interval' => 900,
	'cdn.queue.limit' => 25,
	'cdn.force.rewrite' => false,
	'cdn.autoupload.enabled' => false,
	'cdn.autoupload.interval' => 3600,
	'cdn.canonical_header' => false,
	'cdn.ftp.host' => '',
	'cdn.ftp.user' => '',
	'cdn.ftp.pass' => '',
	'cdn.ftp.path' => '',
	'cdn.ftp.pasv' => false,
	'cdn.ftp.domain' => array(
	),
	'cdn.ftp.ssl' => 'auto',
	'cdn.s3.key' => '',
	'cdn.s3.secret' => '',
	'cdn.s3.bucket' => '',
	'cdn.s3.cname' => array(
	),
	'cdn.s3.ssl' => 'auto',
	'cdn.cf.key' => '',
	'cdn.cf.secret' => '',
	'cdn.cf.bucket' => '',
	'cdn.cf.id' => '',
	'cdn.cf.cname' => array(
	),
	'cdn.cf.ssl' => 'auto',
	'cdn.cf2.key' => '',
	'cdn.cf2.secret' => '',
	'cdn.cf2.id' => '',
	'cdn.cf2.cname' => array(
	),
	'cdn.cf2.ssl' => '',
	'cdn.rscf.user' => '',
	'cdn.rscf.key' => '',
	'cdn.rscf.location' => 'us',
	'cdn.rscf.container' => '',
	'cdn.rscf.cname' => array(
	),
	'cdn.rscf.ssl' => 'auto',
	'cdn.azure.user' => '',
	'cdn.azure.key' => '',
	'cdn.azure.container' => '',
	'cdn.azure.cname' => array(
	),
	'cdn.azure.ssl' => 'auto',
	'cdn.mirror.domain' => array(
	),
	'cdn.mirror.ssl' => 'auto',
	'cdn.netdna.alias' => '',
	'cdn.netdna.consumerkey' => '',
	'cdn.netdna.consumersecret' => '',
	'cdn.netdna.authorization_key' => '',
	'cdn.netdna.domain' => array(
	),
	'cdn.netdna.ssl' => 'auto',
	'cdn.netdna.zone_id' => 0,
	'cdn.maxcdn.authorization_key' => '',
	'cdn.maxcdn.domain' => array(
	),
	'cdn.maxcdn.ssl' => 'auto',
	'cdn.maxcdn.zone_id' => 0,
	'cdn.cotendo.username' => '',
	'cdn.cotendo.password' => '',
	'cdn.cotendo.zones' => array(
	),
	'cdn.cotendo.domain' => array(
	),
	'cdn.cotendo.ssl' => 'auto',
	'cdn.akamai.username' => '',
	'cdn.akamai.password' => '',
	'cdn.akamai.email_notification' => array(
	),
	'cdn.akamai.action' => 'invalidate',
	'cdn.akamai.zone' => 'production',
	'cdn.akamai.domain' => array(
	),
	'cdn.akamai.ssl' => 'auto',
	'cdn.edgecast.account' => '',
	'cdn.edgecast.token' => '',
	'cdn.edgecast.domain' => array(
	),
	'cdn.edgecast.ssl' => 'auto',
	'cdn.att.account' => '',
	'cdn.att.token' => '',
	'cdn.att.domain' => array(
	),
	'cdn.att.ssl' => 'auto',
	'cdn.reject.admins' => false,
	'cdn.reject.logged_roles' => false,
	'cdn.reject.roles' => array(
	),
	'cdn.reject.ua' => array(
	),
	'cdn.reject.uri' => array(
	),
	'cdn.reject.files' => array(
		0 => '{uploads_dir}/wpcf7_captcha/*',
		1 => '{uploads_dir}/imagerotator.swf',
		2 => '{plugins_dir}/wp-fb-autoconnect/facebook-platform/channel.html',
	),
	'cdn.reject.ssl' => false,
	'cdncache.enabled' => false,
	'varnish.enabled' => false,
	'varnish.debug' => false,
	'varnish.servers' => array(
		0 => '',
	),
	'browsercache.enabled' => true,
	'browsercache.no404wp' => false,
	'browsercache.no404wp.exceptions' => array(
		0 => 'robots\\.txt',
		1 => 'sitemap(_index)?\\.xml(\\.gz)?',
		2 => '[a-z0-9_\\-]+-sitemap([0-9]+)?\\.xml(\\.gz)?',
	),
	'browsercache.cssjs.last_modified' => true,
	'browsercache.cssjs.compression' => true,
	'browsercache.cssjs.expires' => true,
	'browsercache.cssjs.lifetime' => 31536000,
	'browsercache.cssjs.nocookies' => false,
	'browsercache.cssjs.cache.control' => true,
	'browsercache.cssjs.cache.policy' => 'cache_public_maxage',
	'browsercache.cssjs.etag' => true,
	'browsercache.cssjs.w3tc' => true,
	'browsercache.cssjs.replace' => false,
	'browsercache.html.compression' => true,
	'browsercache.html.last_modified' => true,
	'browsercache.html.expires' => true,
	'browsercache.html.lifetime' => 3200,
	'browsercache.html.cache.control' => true,
	'browsercache.html.cache.policy' => 'cache_public_maxage',
	'browsercache.html.etag' => true,
	'browsercache.html.w3tc' => true,
	'browsercache.html.replace' => false,
	'browsercache.other.last_modified' => true,
	'browsercache.other.compression' => true,
	'browsercache.other.expires' => true,
	'browsercache.other.lifetime' => 31536000,
	'browsercache.other.nocookies' => false,
	'browsercache.other.cache.control' => true,
	'browsercache.other.cache.policy' => 'cache_public_maxage',
	'browsercache.other.etag' => true,
	'browsercache.other.w3tc' => true,
	'browsercache.other.replace' => false,
	'browsercache.timestamp' => '1399550062',
	'browsercache.replace.exceptions' => array(
		0 => '',
	),
	'mobile.enabled' => true,
	'mobile.rgroups' => array(
		'high' => array(
			'theme' => '',
			'enabled' => true,
			'redirect' => '',
			'agents' => array(
				0 => 'acer\\ s100',
				1 => 'android',
				2 => 'archos5',
				3 => 'bada',
				4 => 'bb10',
				5 => 'blackberry9500',
				6 => 'blackberry9530',
				7 => 'blackberry9550',
				8 => 'blackberry\\ 9800',
				9 => 'cupcake',
				10 => 'docomo\\ ht\\-03a',
				11 => 'dream',
				12 => 'froyo',
				13 => 'googlebot-mobile',
				14 => 'htc\\ hero',
				15 => 'htc\\ magic',
				16 => 'htc_dream',
				17 => 'htc_magic',
				18 => 'iemobile/7.0',
				19 => 'incognito',
				20 => 'ipad',
				21 => 'iphone',
				22 => 'ipod',
				23 => 'kindle',
				24 => 'lg\\-gw620',
				25 => 'liquid\\ build',
				26 => 'maemo',
				27 => 'mot\\-mb200',
				28 => 'mot\\-mb300',
				29 => 'nexus\\ 7',
				30 => 'nexus\\ one',
				31 => 'opera\\ mini',
				32 => 's8000',
				33 => 'samsung\\-s8000',
				34 => 'series60.*webkit',
				35 => 'series60/5\\.0',
				36 => 'sonyericssone10',
				37 => 'sonyericssonu20',
				38 => 'sonyericssonx10',
				39 => 't\\-mobile\\ mytouch\\ 3g',
				40 => 't\\-mobile\\ opal',
				41 => 'tattoo',
				42 => 'touch',
				43 => 'webmate',
				44 => 'webos',
			),
		),
		'low' => array(
			'theme' => '',
			'enabled' => true,
			'redirect' => '',
			'agents' => array(
				0 => '240x320',
				1 => '2\\.0\\ mmp',
				2 => '\\bppc\\b',
				3 => 'alcatel',
				4 => 'amoi',
				5 => 'asus',
				6 => 'au\\-mic',
				7 => 'audiovox',
				8 => 'avantgo',
				9 => 'bb10',
				10 => 'benq',
				11 => 'bird',
				12 => 'blackberry',
				13 => 'blazer',
				14 => 'cdm',
				15 => 'cellphone',
				16 => 'danger',
				17 => 'ddipocket',
				18 => 'docomo',
				19 => 'dopod',
				20 => 'elaine/3\\.0',
				21 => 'ericsson',
				22 => 'eudoraweb',
				23 => 'fly',
				24 => 'haier',
				25 => 'hiptop',
				26 => 'hp\\.ipaq',
				27 => 'htc',
				28 => 'huawei',
				29 => 'i\\-mobile',
				30 => 'iemobile',
				31 => 'iemobile/7',
				32 => 'iemobile/9',
				33 => 'j\\-phone',
				34 => 'kddi',
				35 => 'konka',
				36 => 'kwc',
				37 => 'kyocera/wx310k',
				38 => 'lenovo',
				39 => 'lg',
				40 => 'lg/u990',
				41 => 'lge\\ vx',
				42 => 'midp',
				43 => 'midp\\-2\\.0',
				44 => 'mmef20',
				45 => 'mmp',
				46 => 'mobilephone',
				47 => 'mot\\-v',
				48 => 'motorola',
				49 => 'msie\\ 10\\.0',
				50 => 'netfront',
				51 => 'newgen',
				52 => 'newt',
				53 => 'nintendo\\ ds',
				54 => 'nintendo\\ wii',
				55 => 'nitro',
				56 => 'nokia',
				57 => 'novarra',
				58 => 'o2',
				59 => 'openweb',
				60 => 'opera\\ mobi',
				61 => 'opera\\.mobi',
				62 => 'p160u',
				63 => 'palm',
				64 => 'panasonic',
				65 => 'pantech',
				66 => 'pdxgw',
				67 => 'pg',
				68 => 'philips',
				69 => 'phone',
				70 => 'playbook',
				71 => 'playstation\\ portable',
				72 => 'portalmmm',
				73 => 'proxinet',
				74 => 'psp',
				75 => 'qtek',
				76 => 'sagem',
				77 => 'samsung',
				78 => 'sanyo',
				79 => 'sch',
				80 => 'sch\\-i800',
				81 => 'sec',
				82 => 'sendo',
				83 => 'sgh',
				84 => 'sharp',
				85 => 'sharp\\-tq\\-gx10',
				86 => 'small',
				87 => 'smartphone',
				88 => 'softbank',
				89 => 'sonyericsson',
				90 => 'sph',
				91 => 'symbian',
				92 => 'symbian\\ os',
				93 => 'symbianos',
				94 => 'toshiba',
				95 => 'treo',
				96 => 'ts21i\\-10',
				97 => 'up\\.browser',
				98 => 'up\\.link',
				99 => 'uts',
				100 => 'vertu',
				101 => 'vodafone',
				102 => 'wap',
				103 => 'willcome',
				104 => 'windows\\ ce',
				105 => 'windows\\.ce',
				106 => 'winwap',
				107 => 'xda',
				108 => 'xoom',
				109 => 'zte',
			),
		),
	),
	'referrer.enabled' => true,
	'referrer.rgroups' => array(
		'search_engines' => array(
			'theme' => '',
			'enabled' => true,
			'redirect' => '',
			'referrers' => array(
				0 => 'ask\\.com',
				1 => 'bing\\.com',
				2 => 'google\\.com',
				3 => 'msn\\.com',
				4 => 'yahoo\\.com',
			),
		),
	),
	'common.support' => '',
	'common.install' => 0,
	'common.tweeted' => false,
	'config.check' => true,
	'config.path' => '',
	'widget.latest.items' => 3,
	'widget.latest_news.items' => 5,
	'widget.pagespeed.enabled' => false,
	'widget.pagespeed.key' => '',
	'notes.wp_content_changed_perms' => true,
	'notes.wp_content_perms' => false,
	'notes.theme_changed' => false,
	'notes.wp_upgraded' => false,
	'notes.plugins_updated' => true,
	'notes.cdn_upload' => false,
	'notes.cdn_reupload' => false,
	'notes.need_empty_pgcache' => false,
	'notes.need_empty_minify' => false,
	'notes.need_empty_objectcache' => false,
	'notes.root_rules' => true,
	'notes.rules' => true,
	'notes.pgcache_rules_wpsc' => true,
	'notes.support_us' => true,
	'notes.no_curl' => true,
	'notes.no_zlib' => true,
	'notes.zlib_output_compression' => true,
	'notes.no_permalink_rules' => true,
	'notes.browsercache_rules_no404wp' => true,
	'timelimit.email_send' => 180,
	'timelimit.varnish_purge' => 300,
	'timelimit.cache_flush' => 600,
	'timelimit.cache_gc' => 600,
	'timelimit.cdn_upload' => 600,
	'timelimit.cdn_delete' => 300,
	'timelimit.cdn_purge' => 300,
	'timelimit.cdn_import' => 600,
	'timelimit.cdn_test' => 300,
	'timelimit.cdn_container_create' => 300,
	'timelimit.domain_rename' => 120,
	'timelimit.minify_recommendations' => 600,
	'minify.auto.filename_length' => 150,
	'minify.auto.disable_filename_length_test' => false,
	'common.instance_id' => 2061302123,
	'common.force_master' => true,
	'newrelic.enabled' => false,
	'newrelic.api_key' => '',
	'newrelic.account_id' => '',
	'newrelic.application_id' => 0,
	'newrelic.appname' => '',
	'newrelic.accept.logged_roles' => true,
	'newrelic.accept.roles' => array(
		0 => 'contributor',
	),
	'newrelic.use_php_function' => true,
	'notes.new_relic_page_load_notification' => true,
	'newrelic.appname_prefix' => 'Child Site - ',
	'newrelic.merge_with_network' => true,
	'newrelic.cache_time' => 5,
	'newrelic.enable_xmit' => false,
	'newrelic.use_network_wide_id' => false,
	'pgcache.late_init' => false,
	'newrelic.include_rum' => true,
	'extensions.settings' => array(
		'genesis.theme' => array(
			'wp_head' => '0',
			'genesis_header' => '1',
			'genesis_do_nav' => '1',
			'genesis_do_subnav' => '1',
			'loop_front_page' => '1',
			'loop_single' => '1',
			'loop_single_excluded' => '',
			'loop_single_genesis_comments' => '0',
			'loop_single_genesis_pings' => '0',
			'sidebar' => '0',
			'sidebar_excluded' => '',
			'genesis_footer' => '1',
			'wp_footer' => '0',
			'reject_logged_roles' => '1',
			'reject_logged_roles_on_actions' => array(
				0 => 'genesis_loop',
				1 => 'wp_head',
				2 => 'wp_footer',
			),
			'reject_roles' => array(
				0 => 'administrator',
			),
		),
	),
	'extensions.active' => array(
	),
	'plugin.license_key' => '',
	'plugin.type' => '',
);